tests/xrt/100_ert_ncu/xrtxx-ip.cpp

```diff
@@ -47,6 +47,7 @@ static constexpr uint32_t AP_START    = 0x1;
 static constexpr uint32_t AP_DONE     = 0x2;
 static constexpr uint32_t AP_IDLE     = 0x4;
 static constexpr uint32_t AP_CONTINUE = 0x10;
+static constexpr uint32_t ADDR_ISR = 0x0C;
```

Comment:
Format issue: Align to the above equal values '='
```suggestion
static constexpr uint32_t ADDR_ISR   = 0x0C;
```

Reply:
done